Github/service.js

```diff
@@ -101,9 +101,9 @@ const getBranches = branchObject => {
         url: branchObject.url,
         headers: { 'User-Agent': 'request' },
       }, (err, res, data) => {
-        if (err) {
-          console.log('Error:', err);
-          reject(err);
+        if (res.statusCode !== 200) {
+          console.log('Error:', res.statusMessage);
+          reject(new Error(`Unable to get repos.`));
```

Comment:
nitpick: you are repeating `Unable to get repos.` msg in every function, would be useful to know if error happened in `get Commits`, `getRepos` or `getBranches` etc

Reply:
fixed